File: Phase - 7 Automation Testing/Assessment-3/cucumber/src/test/java/pages/LoginPage.java
package pages;

import org.openqa.selenium.By;

public class LoginPage {
	public By email = By.xpath("//input[@name = \"email\"]");
	public By password = By.xpath("//input[@name = \"password\"]");
	public By loginButton = By.xpath("//input[@value = \"Login\"]");
	
}	

File: Phase - 9 Automation Testing Using Python/Pytest Assessment/TutorialNinja/pages/login_page.py
from selenium.webdriver.common.by import By

class LoginPage:
    email_input = (By.ID, "input-email")
    password_input = (By.ID, "input-password")
    login_button = (By.XPATH,"//input[@value='Login']")
    error_message = (By.XPATH, "//div[contains(@class ,\"alert\")]")
    